src/ImageRunModal.jsx

```diff
@@ -908,6 +931,46 @@ export class ImageRunModal extends React.Component {
                             <Checkbox isChecked={this.state.runImage} id="run-image-dialog-start-after-creation"
                                       onChange={value => this.onValueChanged('runImage', value)} label={_("Start after creation")} />
                         </FormGroup>
+                        <Grid hasGutter md={6}>
+                            { this.props.userServiceAvailable && this.props.systemServiceAvailable &&
+                            <GridItem span={dialogValues.restartPolicy === "on-failure" ? 6 : 12}>
```

Comment:
This logic behaves a bit weirdly on mobile:
![Screen Shot 2021-12-14 at 09 16 36](https://user-images.githubusercontent.com/12330670/145959798-edbef054-874e-49ec-b074-19387c2428de.png)

I think we want the first select always be about half of the screen. I think we can do something like `max-width: 70px` or something - that way it is not through the whole dialog just for `No`. And then let PF doing its wrapping on mobile.

Reply:
> max-width: 70px

Never use arbitrary pixels for things that contain text. It should be sized via PatternFly, CSS variables, or CSS units that are based on font size (or in some cases, screen size).

This layout is lacking breakpoints for whatever reason. 

In this case, it's because the gridItem has variable spanning that's hardcoded without respect to mobile, even though the grid itself is using md.

If we were to use PF sizing, it would need breakpoints. Something like `pf-u-w-50` would be `pf-u-w-50-on-md` (for example, although the size and breakpoint varies).